app/auth/views/users.py

```diff
@@ -250,28 +250,32 @@ def submit_create_account():
 def send_reset_password_email():
     json_payload = get_json_from_request()
     email_address = json_payload.get('email_address', None)
+    framework = json_payload.get('framework', 'digital-marketplace')
```

Comment:
Don't like embedding parameters in post body.  

I would go with `/reset-password/<string:token>/framework/<string:framework-slug>`

or 

`/reset-password/framework/<string:framework-slug>/token/<string:token>`